modules/standard/BigInteger.chpl

```diff
@@ -381,6 +381,9 @@ module BigInteger {
   //
   // Cast operators
   //
+  proc _cast(type toType: bigint, src) {
```

Comment:
Seems like `src` should have a particular type restriction, for example `integral` is probably what is intended here.

Reply:
A `bigint` can also be created from a string, so we might want to allow this as well:

```chpl
 var y = "4847382292989382987395534934347": bigint;
```

I could add a separate cast for `string` in addition to the `integral` one.